### DISCUSSION TITLE
using the pydantic logo?

### DISCUSSION THREAD

**Discussion by jgunstone** [pydantic/pydantic#8325] [Question] [answered]
hi there, 
I created a package [ipyautoui](https://github.com/maxfordham/ipyautoui) which uses pydantic models to create user input forms using [ipywidgets](https://ipywidgets.readthedocs.io/en/stable/). 

I created a logo, which incorporates the pydantic logo.... 
[https://github.com/maxfordham/ipyautoui/blob/main/docs/images/logo.png](https://github.com/maxfordham/ipyautoui/blob/main/docs/images/logo.png)
is it ok for me to use the pydantic logo in this way ? 

on reflection perhaps you'd prefer it if I didn't as this isn't a "pydantic-supported" package...

**Comment by samuelcolvin** [comment, marked as answer]
Sounds good, thanks so much for using Pydantic and asking.

Would probably be good if you call out in the readme that the package is not part of pydantic

**Reply by jgunstone** [reply]
ok will do - thanks!
